app/helpers/view_components/profile/header.rb

```diff
@@ -23,6 +24,14 @@ def top_three_tracks
         ::Track.where(id: track_ids).
           order(Arel.sql("FIND_IN_SET(id, '#{track_ids.join(',')}')"))
       end
+
+      def header_tags
+        tags = []
+        tags << { class: "tag staff", icon: :logo, title: "Exercism Staff" } if @user.staff?
+        tags << { class: "tag maintainer", icon: :maintaining, title: "Maintainer" } if @user.maintainer?
+        tags << { class: "tag maintainer", icon: :insiders, title: "Insider" } if %i[eligible eligible_lifetime].include?(@user.insiders_status) # rubocop:disable Layout/LineLength
```

Comment:
```suggestion
        tags << { class: "tag maintainer", icon: :insiders, title: "Insider" } if %i[active active_lifetime].include?(@user.insiders_status) # rubocop:disable Layout/LineLength
```

Please also add a test for this.